Neuro/Minigames/Completion/Solvers/FixLightsSolver.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using Neuro.Cursor;
+
+namespace Neuro.Minigames.Completion.Solvers;
+
+[MinigameSolver(typeof(SwitchMinigame))]
+public sealed class FixLightsSolver : MinigameSolver<SwitchMinigame>
+{
+    public override IEnumerator CompleteMinigame(SwitchMinigame minigame, NormalPlayerTask task)
+    {
+        for (int i = 0; i < minigame.switches.Count; i++)
```

Comment:
Other players can turn off switches so this logic should be replaced with something like:

while sabotage isn't fixed
\-    pick the closest switch to the cursor which if off
\-    click it

You can check ClearAsteroidsSolver.cs because that also has similar logic